scripts/skills/actives/rf_net_pull_skill.nut

```diff
@@ -55,6 +55,18 @@ this.rf_net_pull_skill <- ::inherit("scripts/skills/skill", {
 			icon = "ui/icons/special.png",
 			text = ::Reforged.Mod.Tooltips.parseString("The target will lose the [Shieldwall|Skill+shieldwall_effect], [Spearwall+Skill+spearwall_effect] and [Riposte|Skill+riposte_effect] effects")
 		});
+		ret.push({
+			id = 15,
+			type = "text",
+			icon = "ui/icons/special.png",
+			text = "Will expend the currently held net"
+		});
+		ret.push({
+			id = 20,
+			type = "text",
+			icon = "ui/icons/warning.png",
+			text = "There must be an open tile for the target to be pulled into"
```

Comment:
This open tile entry is redundant imo (i.e. it is obvious from the use case of the skill) and should be removed.